### PR TITLE
🐛 Activate button above content.

### LINKED ISSUE #30420: "activate 360" button is hidden by cta layer
#30381 did not resolve this.

`i-amphtml-story-360-activate-button` is currently a descendant of `amp-story-grid-layer` which has `z-index: 2`.
`amp-story-cta-layer` has a `z-index: 3` so it will always be covered.

This element might need to be attached at the `amp-story` or `amp-story-page` level for it to be above content.

Screen Shot 2020-09-28 at 1 03 50 PM

cc @ampproject/wg-stories @t0mg

### PR DESCRIPTION
**Pull request by processprocess**
Context / fixes #30420
Screen Shot 2020-10-01 at 1 22 03 PM

### CODE REVIEW

File: extensions/amp-story-360/0.1/amp-story-360.js [hunk 1]
```diff
@@ -338,23 +341,33 @@ export class AmpStory360 extends AMP.BaseElement {
...
 
+  /**
+   * @private
+   * @return {!Element} the parent amp-story-page
+   */
+  getPage_() {
+    return closest(dev().assertElement(this.element), (el) => {
+      return el.tagName.toLowerCase() === 'amp-story-page';
+    });
+  }
```
Comment (kristoferbaxter) [inline extensions/amp-story-360/0.1/amp-story-360.js:357]: Nit: Can just use the builtin return. 

```suggestion
  getPage_() {
    return closest(dev().assertElement(this.element), (el) => 
      el.tagName.toLowerCase() === 'amp-story-page';
    );
  }
```

**Review by kristoferbaxter — COMMENTED** [review]
Looks good to me, but left a comment about the method return value.

**Comment by kristoferbaxter** [comment]
CLA violation cleared (It was due to accepting merge suggestion via Github UI).

**Review by gmajoulet — APPROVED** [review]

File: extensions/amp-story-360/0.1/amp-story-360.js [hunk 2]
```diff
@@ -338,23 +341,36 @@ export class AmpStory360 extends AMP.BaseElement {
...
     return this.pageId_;
   }
 
+  /**
+   * @private
+   * @return {!Element} the parent amp-story-page
+   */
+  getPage_() {
+    return closest(dev().assertElement(this.element), (el) => 
+      el.tagName.toLowerCase() === 'amp-story-page';
```
Comment (kristoferbaxter) [inline extensions/amp-story-360/0.1/amp-story-360.js:355]: Think I made this mistake in earlier suggestion.

```suggestion
      el.tagName.toLowerCase() === 'amp-story-page'
```

**Pull request merged**